odoo/models.py

```diff
@@ -74,11 +74,18 @@
 regex_pg_name = re.compile(r'^[a-z_][a-z0-9_$]*$', re.I)
 regex_field_agg = re.compile(r'(\w+)(?::(\w+)(?:\((\w+)\))?)?')
 
+regex_aggregate_spec = re.compile(r'(\w+)(?::(\w+))?$')
 AUTOINIT_RECALCULATE_STORED_FIELDS = 1000
 
 INSERT_BATCH_SIZE = 100
 SQL_DEFAULT = psycopg2.extensions.AsIs("DEFAULT")
 
+def split_read_group_spec(spec: str) -> 'tuple[str]':
+    res_match = regex_aggregate_spec.match(spec)
+    if not res_match:
+        raise ValueError(f'Invalid aggregate/groupby specification {spec!r}. Valid specification example: "field:agg"')
```

Comment:
can't we translate the error message here ? 
(same below)

Reply:
IMO, we should never translate developer error. User should never see it (because he cannot construct it by itself).